skrub/tests/test_table_vectorizer.py

```diff
@@ -595,3 +619,29 @@ def test__infer_date_format() -> None:
     # Test with a column containing more than two date formats
     date_column = pd.Series(["2022-01-01", "01/02/2022", "20220103", "2022-Jan-04"])
     assert _infer_date_format(date_column) is None
+
+
+def test_mixed_types():
+    # TODO: datetime/str mixed types
+    # don't work
```

Comment:
Is this TODO fixed?

Reply:
no, it seems to require a bit more work, and I was wondering whether such mixed types could really appear in the wild